html/js/guiutils/Tools.js

```diff
@@ -6,6 +6,7 @@ GTE.UI = (function (parentModule) {
     * @class
     */
     function Tools() {
+        this.activePlayer = -1;
```

Comment:
On the newTree() function, you have to remove all the players (the first three will get re-created when calling the Tree() constructor), and to assign activePlayer a value of -1. I would also switch mode to ADD